lib/console.ex

```diff
@@ -6,4 +6,33 @@ defmodule Console do
   def display(message) do
     IO.puts(message)
   end
+
+  def format_board([]), do: []
+
```

Comment:
I am wondering if formatting the board is the responsibility of the console. Maybe this is something you can extract out.